app/controllers/concerns/default_url_options.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module DefaultUrlOptions
+  extend ActiveSupport::Concern
+
+  class_methods do
+    # FIXME(ezekg) Why is this needed?
+    def default_url_options(...) = Rails.application.default_url_options(...)
```

Comment:
Why is this needed? And why was the previous way of doing this failing only when the `CI` environment variable is set?!